src/main/flight/navigation_rewrite_multicopter.c

```diff
@@ -483,11 +486,19 @@ bool isMulticopterLandingDetected(uint32_t * landingTimer)
     // from processRx() and rcCommand at that moment holds rc input, not adjusted values from NAV core)
     bool minimalThrust = rcCommandAdjustedThrottle < posControl.navConfig->mc_min_fly_throttle;
 
+    /*
+    Do not trust the sonar, it wants to kill you!
```

Comment:
Sonar is ok, we just need a better way to detect if its healthy and can be trusted.